kolibri/plugins/epub_viewer/assets/src/views/AddEditCustomThemeModal.vue

```diff
@@ -224,6 +224,9 @@
         }
         this.showColorPicker = null;
       },
+      generateAriaLabel(color) {
```

Comment:
Hi @Akila-I - after some chatting with @radinamatic, I think this function will be too complicated for localization purposes (even though it does present the correct text). It will be a bit tricky to explain to translators how to properly managed this through the platform we use. Instead of using a function here, let's just add the extra strings:
'Select link color', 'Select background color', etc. 
And reference them directly

Reply:
done